content/md/products-super-power/enrich-your-reference-entity-records.md

```diff
@@ -0,0 +1,142 @@
+
+::: info
+In the records grid, you will see only certain types of attributes: code, text and image.
+:::
+
+::: info
+When you scroll down in the grid, only the first 500 results are displayed (in order to avoid performances issues). When you arrive at the end of the first 500 records, a message is displayed *"You just scrolled through the first 500 results out of 10005 records. To see more, try with more specific search criteria."*
+:::
+
+Of course, you can still edit or delete a record thanks to the buttons at the end of each line.
+
+## Search for a record
+
+### Full text search
+
+Our goal is to facilitate your work and to **increase your productivity**. To make this possible, we made a very powerful search in the grid of records. Indeed, we implemented a **full text search** which allows you to search on code, label and text attribute types.
+
+For example, if you search for the word *« famous »*, all the records containing the word *« famous »* in their code, label or text attribute types will be displayed.  
+
+![Search for records with a keyword](../img/Records_Search_1_word.png)
+
+And we can go further.
+If you search for *"famous"* and *"bags"*, then only the record containing both *"famous"* and *"bags"* words will be displayed.
+
+![Search for records with several keywords](../img/Records_Search_2_words.png)
+
+### Filter on completeness
+
+You can also filter on completeness in order to see the records that are complete/incomplete, or choose to see all the records, no matter their completeness.
+
+![Records filter on completeness](../img/Records_filter_completeness.png)
+
+::: tips
+To know how to define the required attributes for the completeness, please take a look at this [article](/articles/manage-reference-entities.html)
+:::
+
+## Edit a record
+
+Of course, you can edit a record in order to change values or add new ones. Nothing simpler: just go on a record and add/change the values you want.
+
+As for a product, you can define if you want a **different content according to channels**, and same for **locales**.
+You can find a **locale and channel switcher** on each record, meaning that if you need to translate values for example, you can do it simply switching from a locale to another.
+
+![Locale switcher on the records grid](../img/Records_locale_switcher.png)
+
+In the Attribute tab, if you defined a *"text area"* attribute for your records, you can tick the parameter `rich text editor`.   This will give you access to a set of options to improve the formate of your text. For example, you can change the font, the size, etc. You can also add a link, a video, an image.
+In fact, the Rich text editor offers a large bunch of possibilities!
+
+![Records text area](../img/Records_text_area.png)
+
+
+## Follow the records completeness
+
+As in the product, we calculate the completeness on records.
+
+::: tips
```

Comment:
This tips is the same as the one few line above, is it useful to add it once again?
Maybe, add a link on the completeness word above pointing to /articles/manage-reference-entities.html